util/topgen/intermodule.py

```diff
@@ -251,19 +251,24 @@ def elab_intermodule(topcfg: OrderedDict):
                              ('width', sig["width"]), ('type', sig["type"]),
                              ('default', sig["default"])]))
 
-    if "external" not in topcfg["inter_module"]:
-        topcfg["inter_module"]["external"] = []
-        topcfg["inter_signal"]["external"] = []
+    topcfg["inter_module"].setdefault('external', [])
+    topcfg["inter_signal"].setdefault('external', [])
 
-    if "external" not in topcfg["inter_signal"]:
-        topcfg["inter_signal"]["external"] = []
-
-    for s in topcfg["inter_module"]["external"]:
+    for s, ts in topcfg["inter_module"]["external"].items():
```

Comment:
is `ts` supposed to stand for top signal here? I kind of wonder in the case of `external` it should just be port. 
anyways, this is just a nit.

Reply:
Changed to `port` :)